docs/pom.xml

```diff
@@ -78,6 +78,11 @@
             <groupId>io.smallrye.stork</groupId>
             <artifactId>stork-service-discovery-static-list</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye.stork</groupId>
+            <artifactId>stork-spring-boot-config</artifactId>
+<!--            <scope>provided</scope>-->
```

Comment:
Why is this commented out?

Reply:
Just testing something and forgot to un-comment then